package.ts

```diff
@@ -0,0 +1,16 @@
+import { build } from 'esbuild';
+
+(async () => {
+  const zipName = process.env.ZIP_NAME || 'authoriser';
```

Comment:
Might be a good idea to put this in the common lib to reduce duplication of this function as it's the same as what is in the logs repo

Reply:
There is already a bundling method in the common lib, but the structure of this repo does not currently lend itself to make use of that, hence this independent solution.

Fair point though and definitely one to consider when this repo is refactored/implemented using the new template.

Reply:
Personally, I'm not sure I see the need to move this logic to a common repo.

The current shared `@dvsa/service-bundler` has a far more logic. The bundler package has code to traverse the file structure, building for Proxies (not applicable here) and Functions, copying non TS files into the build (e.g. if you wanted to include XML/YAML), programatic zipping renaming of artefacts etc. That largely relies on the new template folder structure.
All of that is ~300 lines of code, so that's why thats been centralised.

This is a far more simplistic use case and therefore we probably don't need to complicate matters would be my take.

Even if this was packaged up, we'd still end up here with something like

```
import { readFileSync } from 'node:fs';
import { ServicePackager } from '@dvsa/service-bundler';
import { name, version } from './package.json';

(async () => {
	const [majorNodeVersion] = readFileSync('.nvmrc', 'utf-8').trim().split('.');

	await new ServicePackager({
		nodeMajorVersion: majorNodeVersion,
		handlerFileName: 'index.ts',
	}).build({
		zip: process.argv.includes('--package'),
	});
})();
```